src/app/embed/embed.component.css

```diff
@@ -1,7 +1,7 @@
 play-player {
   position: absolute;
   width: 100%;
-  height: 185px;
+  height: 100%;
```

Comment:
basically all of the flexibility I added was just to handle the cases right now where crappy embeds get "maintain aspect ratio" enabled, as was the case with the songexploder embed I saw floating around a while back and the scrubber got cut off. So really just looking to get enough flex in place so that those embeds look ok.

beyond that I don't really care, up to you